src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square() {
+    }
+
+    public Square(double side) {
+        this.typeOfFigure = TypeOfFigure.SQUARE;
+        this.area = getQuadrangularArea(side, side);
```

Comment:
All magic numbers in your code should be constants. Here, the value for the area calculation should be defined as a constant if it's a fixed value or calculated dynamically if it's supposed to change.